nodes.py

```diff
@@ -174,16 +179,100 @@ def run(self, image: Tensor, multiple_of: int, method: str) -> Tuple[Tensor]:
             return (image[:, top:bottom, left:right, :],)
 
 
+class HFHubLoraLoader:
+    def __init__(self):
+        self.loaded_lora = None
+
+    @classmethod
+    def INPUT_TYPES(s):
+        return {
+            "required": {
+                "model": ("MODEL",),
+                "clip": ("CLIP",),
+                "repo_id": (
+                    "STRING",
+                    {
+                        "default": ""
+                    }
+                ),
+                "subfolder": (
+                    "STRING",
+                    {
+                        "default": ""
+                    }
+                ),
+                "filename": (
+                    "STRING",
+                    {
+                        "default": ""
+                    }
+                ),
+                "strength_model": (
+                    "FLOAT", {
+                        "default": 1.0,
+                        "min": -20.0,
+                        "max": 20.0,
+                        "step": 0.01
+                    }
+                ),
+                "strength_clip": (
+                    "FLOAT",
+                    {
+                        "default": 1.0,
+                        "min": -20.0,
+                        "max": 20.0,
+                        "step": 0.01
+                    }
+                ),
+            }
+        }
+
+    RETURN_TYPES = ("MODEL", "CLIP")
+    FUNCTION = "load_lora"
+
+    CATEGORY = "loaders"
+
+    def load_lora(self, model, clip, repo_id: str, subfolder: str, filename: str, strength_model: float,
+                  strength_clip: float):
+        if strength_model == 0 and strength_clip == 0:
+            return (model, clip)
+
+        lora_path = hf_hub_download(
+            repo_id=repo_id.strip(),
+            subfolder=None if subfolder is None or subfolder.strip() == "" else subfolder.strip(),
+            filename=filename.strip(),
+            cache_dir=find_or_create_cache(),
```

Comment:
Since it's a lora and comfy has a known folder for that, maybe use hf_hub_download's local_dir arg to point to the comfy lora dir?

Reply:
We could, but hf_hub_download doesnt make a clean file download. It is their lock/symlink/hashed cache folder structure. I didn't want to dirty the lora folder with the model cache data.
![Screenshot from 2024-04-16 07-11-02](https://github.com/glifxyz/ComfyUI-GlifNodes/assets/195960/b38380f4-be13-45b5-be34-f06852b2ded1)

Reply:
use `local_dir_use_symlinks=False,`?

Reply:
That would work if we also define the LoRA dir as `local_dir` screenshot below of doing it that way in the a `lora` subfolder. HF still needs a place for the cache, but it can copy the actual LoRA to the lora folder when done like this. One problem we may run into doing it this way is handling diffusers based LoRAs that all face the filename `pytorch_lora_weights.safetensors`. I believe they would overwrite each other or it would pull the wrong one. We can force the filename if it is `pytorch_lora_weights.safetensors` to a generated one based on the repo name. 

![Screenshot from 2024-04-16 08-50-34](https://github.com/glifxyz/ComfyUI-GlifNodes/assets/195960/6623d511-57a1-4f90-9839-f93d6f820076)